ORBIT/phases/design/electrical_export.py

```diff
@@ -348,16 +372,16 @@ def calc_num_substations(self):
         """
 
         # HVAC substation capacity
-        _substation_capacity = self.offshore_substation_design.get(
+        _substation_capacity = self._oss_design.get(
             "substation_capacity", 1200
```

Comment:
Shouldn't this be a default design parameter as well?

Reply:
Future plans